apps/minifront/src/state/ibc-in.ts

```diff
@@ -109,6 +111,28 @@ const getCompatibleBech32 = (chainName: string, address: Address): string => {
   return bech32Chains.includes(chainName) ? bech32CompatAddress(address) : bech32mAddress(address);
 };
 
+const estimateFee = async ({
+  chainId,
+  client,
+  signerAddress,
+  message,
+}: {
+  chainId: string;
+  client: SigningStargateClient;
+  signerAddress: string;
+  message: EncodeObject;
+}) => {
+  const feeToken = chains.find(({ chain_id }) => chain_id === chainId)?.fees?.fee_tokens[0];
```

Comment:
can there be several `feeTokens`? can there be a case when among 2 `feeTokens` we choose the one which is not on the user's balance?

Reply:
In theory yes. However, in practice, the cosmos chain registry for the chains we are connected to all have one. We may need to revisit this later if we connect to a chain and need to build in logic for choosing a gas token.